Libplanet/Blockchain/Renderers/DelayedActionRenderer.cs

```diff
@@ -244,6 +253,27 @@ public void RenderBlockEnd(Block<T> oldTip, Block<T> newTip)
 
             _localRenderBuffer.Value = new Dictionary<HashDigest<SHA256>, List<ActionEvaluation>>();
             DiscoverBlock(oldTip, newTip);
+
+            if (_reorgResistantHeight == 0)
+            {
+                return;
+            }
+
+            var quotient = Math.DivRem(newTip.Index, _reorgResistantHeight, out _);
```

Comment:
Is it okay that we clear all buffers before threshold(`newTip.Index - _reorgResistantHeight`) instead of dividing?

Reply:
Yes, that's okay. In this case, there is at least one checkpoint block in the buffer, so there is no case where the ActionEvaluation after the checkpoint block does not exist when a reorg occurs.